### PR TITLE
feat(documentation): add styles getting started page

### PR DESCRIPTION
**Pull request by oliverschuerch**

### CODE REVIEW

File: packages/documentation/src/stories/get-started/styles.stories.mdx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+### Available bundles
+
+| Bundle name | Bundle extensions | Description                                                        |
+| :---------- | :---------------- | :----------------------------------------------------------------- |
+| `index`     | `css`, `scss`     | everything for internet facing applications                        |
+| `intranet`  | `css`, `scss`     | everything for internal applications                               |
+| `basics`    | `css`, `scss`     | atomic styles for font, buttons, lists, etc. (no component styles) |
+
+### The core
```
Comment (gfellerph) [inline packages/documentation/src/stories/get-started/styles.stories.mdx:43]:
```suggestion
### Variables, mixins, functions and placeholders
```

File: packages/documentation/src/stories/get-started/styles.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+If you want to use our SCSS core for your custom styles, we recommend to import and use it as follows, to prevent conflicts:
```
Comment (gfellerph) [inline packages/documentation/src/stories/get-started/styles.stories.mdx:45]:
```suggestion
Use the SCSS core for your custom styles and make sure you're always using the most up to date definitions:
```

File: packages/documentation/src/stories/get-started/styles.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+<p class="alert alert-info alert-md">
+  NOTE: The core file itself, does not produce any CSS output. It only contains variables, functions
+  and mixins.
+</p>
+
+```scss dark
+@use '@swisspost/design-system-styles/core.scss' as post;
+
+.my-component {
+  background-color: post.$white;
```
Comment (gfellerph) [inline packages/documentation/src/stories/get-started/styles.stories.mdx:56]:
```suggestion
  background-color: post.$yellow; // #FFCC00
```

File: packages/documentation/src/stories/get-started/styles.stories.mdx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+@use '@swisspost/design-system-styles/<bundle-name>.<extension>';
+```
+
+### Available bundles
+
+| Bundle name | Bundle extensions | Description                                                        |
+| :---------- | :---------------- | :----------------------------------------------------------------- |
+| `index`     | `css`, `scss`     | everything for internet facing applications                        |
+| `intranet`  | `css`, `scss`     | everything for internal applications                               |
+| `basics`    | `css`, `scss`     | atomic styles for font, buttons, lists, etc. (no component styles) |
```
Comment (gfellerph) [inline packages/documentation/src/stories/get-started/styles.stories.mdx:41]:
```suggestion
| `basics`    | `css`, `scss`     | atomic styles for font, buttons, lists, etc. (no component styles) |
| `core`      | `scss`              | variables, mixins, functions and placeholders (no CSS output) |
```

File: packages/documentation/src/stories/get-started/styles.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+      canvas: {
+        hidden: true,
+      },
+    },
+    viewMode: 'docs',
+    badges: [BADGE.NEEDS_REVISION],
+  }}
+/>
+
+# @swisspost/design-system-styles
```
Comment (gfellerph) [inline packages/documentation/src/stories/get-started/styles.stories.mdx:17]:
```suggestion
<h1>@swisspost/design-system-styles<br><span class="fw-light">Styles for the Swiss Post web platform.</span></h1>
```

**Review by gfellerph — CHANGES_REQUESTED** [review]
Nice and easy intro!

File: packages/documentation/src/stories/get-started/styles.stories.mdx (hunk 4 above)
Comment (oliverschuerch) [reply packages/documentation/src/stories/get-started/styles.stories.mdx:41]: The core is only used to write custom styles and therefore no entry file.
That's why I described the usage of it in its own subsection.

File: packages/documentation/src/stories/get-started/styles.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+      },
+    },
+    viewMode: 'docs',
+    badges: [BADGE.NEEDS_REVISION],
+  }}
+/>
+
+<h1>@swisspost/design-system-styles<br><span class="fw-light">Styles for the Swiss Post web platform.</span></h1>
+
+[![npm version](https://badge.fury.io/js/@swisspost%2Fdesign-system-styles.svg)](https://badge.fury.io/js/@swisspost%2Fdesign-system-styles)
```
Comment (gfellerph) [inline packages/documentation/src/stories/get-started/styles.stories.mdx:19]: Broke the heading...

```suggestion
[![npm version](https://badge.fury.io/js/@swisspost%2Fdesign-system-styles.svg)](https://badge.fury.io/js/@swisspost%2Fdesign-system-styles)

# @swisspost/design-system-styles

CSS Styles for the Swiss Post web platform.
```

**Review by gfellerph — CHANGES_REQUESTED** [review]

File: packages/documentation/src/stories/get-started/styles.stories.mdx (hunk 6 above)
Comment (oliverschuerch) [reply packages/documentation/src/stories/get-started/styles.stories.mdx:19]: Have not seen it either!

**Review by alizedebray — APPROVED** [review]

**Review by gfellerph — APPROVED** [review]

**Pull request merged**